### DISCUSSION TITLE
Cant run dev after upgrading to 10.0.1

### DISCUSSION THREAD

**Discussion by PierreVelluet** [vercel/next.js#19147] [Help]
# Bug report
after upgrading next to 10.0.1, npm run dev throw this error : 

`> cars_pro@0.1.0 dev /Users/zotcar/Documents/Dev/cars_pro
> next dev

Loaded env from /Users/zotcar/Documents/Dev/cars_pro/.env
ready - started server on http://localhost:3000
warn  - Your project has both `sass` and `node-sass` installed as dependencies, but should only use one or the other. Please remove the `node-sass` dependency from your project.  Read more: https://err.sh/next.js/duplicate-sass
TypeError: Cannot read property 'concat' of undefined
    at Object.webpack (/Users/zotcar/Documents/Dev/cars_pro/node_modules/next-transpile-modules/src/next-transpile-modules.js:102:73)
    at getBaseWebpackConfig (/Users/zotcar/Documents/Dev/cars_pro/node_modules/next/dist/build/webpack-config.js:136:360)
    at async Promise.all (index 0)
    at async HotReloader.start (/Users/zotcar/Documents/Dev/cars_pro/node_modules/next/dist/server/hot-reloader.js:14:2403)
    at async DevServer.prepare (/Users/zotcar/Documents/Dev/cars_pro/node_modules/next/dist/server/next-dev-server.js:15:414)
    at async /Users/zotcar/Documents/Dev/cars_pro/node_modules/next/dist/cli/next-dev.js:22:1
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! cars_pro@0.1.0 dev: `next dev`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the cars_pro@0.1.0 dev script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/zotcar/.npm/_logs/2020-11-13T14_09_51_774Z-debug.log`

## Describe the bug
I just upgraded to next js 10.0.1 and can no longer start my app with npm run dev.

What i tried : delete node_modules, package.lock, then run npm install and npm run dev.

## Expected behavior

Project start on localhost:3000

## System information

- OS: [MacOS 10.13.6]
- Browser  [chrome]
- Version of Next.js: [10.0.1]
- Version of Node.js: [17.0.1]

**Comment by kachkaev** [comment]
From your error log:

> Your project has both sass and node-sass installed as dependencies, but should only use one or the other. Please remove the node-sass dependency from your project. Read more: https://err.sh/next.js/duplicate-sass

**Comment by PierreVelluet** [comment]
@kachkaev i indeed removed node-sass dependency from my project, but i still have the same error message when i npm run dev :
` cars_pro@0.1.0 dev /Users/zotcar/Documents/Dev/cars_pro
next dev

Loaded env from /Users/zotcar/Documents/Dev/cars_pro/.env
ready - started server on http://localhost:3000
TypeError: Cannot read property 'concat' of undefined
    at Object.webpack (/Users/zotcar/Documents/Dev/cars_pro/node_modules/next-transpile-modules/src/next-transpile-modules.js:102:73)
    at getBaseWebpackConfig (/Users/zotcar/Documents/Dev/cars_pro/node_modules/next/dist/build/webpack-config.js:136:360)
    at async Promise.all (index 0)
    at async HotReloader.start (/Users/zotcar/Documents/Dev/cars_pro/node_modules/next/dist/server/hot-reloader.js:14:2403)
    at async DevServer.prepare (/Users/zotcar/Documents/Dev/cars_pro/node_modules/next/dist/server/next-dev-server.js:15:414)
    at async /Users/zotcar/Documents/Dev/cars_pro/node_modules/next/dist/cli/next-dev.js:22:1
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! cars_pro@0.1.0 dev: `next dev`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the cars_pro@0.1.0 dev script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/zotcar/.npm/_logs/2020-11-16T05_32_12_352Z-debug.log`

My complete log is : 
`0 info it worked if it ends with ok
1 verbose cli [
1 verbose cli   '/Users/zotcar/.nvm/versions/node/v14.15.0/bin/node',
1 verbose cli   '/Users/zotcar/.nvm/versions/node/v14.15.0/bin/npm',
1 verbose cli   'run',
1 verbose cli   'dev'
1 verbose cli ]
2 info using npm@6.14.8
3 info using node@v14.15.0
4 verbose run-script [ 'predev', 'dev', 'postdev' ]
5 info lifecycle cars_pro@0.1.0~predev: cars_pro@0.1.0
6 info lifecycle cars_pro@0.1.0~dev: cars_pro@0.1.0
7 verbose lifecycle cars_pro@0.1.0~dev: unsafe-perm in lifecycle true
8 verbose lifecycle cars_pro@0.1.0~dev: PATH: /Users/zotcar/.nvm/versions/node/v14.15.0/lib/node_modules/npm/node_modules/npm-lifecycle/node-gyp-bin:/Users/zotcar/Documents/Dev/cars_pro/node_modules/.bin:/Users/zotcar/.nvm/versions/node/v14.15.0/bin:/Applications/MAMP/bin/php/php7.2.22/bin:/usr/local/bin:/usr/bin:/bin:/usr/sbin:/sbin:/Applications/MAMP/bin/php/php7.2.22/bin
9 verbose lifecycle cars_pro@0.1.0~dev: CWD: /Users/zotcar/Documents/Dev/cars_pro
10 silly lifecycle cars_pro@0.1.0~dev: Args: [ '-c', 'next dev' ]
11 silly lifecycle cars_pro@0.1.0~dev: Returned: code: 1  signal: null
12 info lifecycle cars_pro@0.1.0~dev: Failed to exec dev script
13 verbose stack Error: cars_pro@0.1.0 dev: `next dev`
13 verbose stack Exit status 1
13 verbose stack     at EventEmitter.<anonymous> (/Users/zotcar/.nvm/versions/node/v14.15.0/lib/node_modules/npm/node_modules/npm-lifecycle/index.js:332:16)
13 verbose stack     at EventEmitter.emit (events.js:315:20)
13 verbose stack     at ChildProcess.<anonymous> (/Users/zotcar/.nvm/versions/node/v14.15.0/lib/node_modules/npm/node_modules/npm-lifecycle/lib/spawn.js:55:14)
13 verbose stack     at ChildProcess.emit (events.js:315:20)
13 verbose stack     at maybeClose (internal/child_process.js:1048:16)
13 verbose stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:288:5)
14 verbose pkgid cars_pro@0.1.0
15 verbose cwd /Users/zotcar/Documents/Dev/cars_pro
16 verbose Darwin 17.7.0
17 verbose argv "/Users/zotcar/.nvm/versions/node/v14.15.0/bin/node" "/Users/zotcar/.nvm/versions/node/v14.15.0/bin/npm" "run" "dev"
18 verbose node v14.15.0
19 verbose npm  v6.14.8
20 error code ELIFECYCLE
21 error errno 1
22 error cars_pro@0.1.0 dev: `next dev`
22 error Exit status 1
23 error Failed at the cars_pro@0.1.0 dev script.
23 error This is probably not a problem with npm. There is likely additional logging output above.
24 verbose exit [ 1, true ]
`

I ran => rm -rf node_modules && rm ./package-lock.json && npm install
I also restarted my computer and MAMP servers.

Any suggestion ?

**Comment by DurandSacha** [comment]
if you have .env.example file, try to copy this in .env or .env.local

`cp .env.example .env`
